packages/fluence.lib.services/src/services/transports/managed.blockchain.transport.service.ts

```diff
@@ -64,7 +64,15 @@ export abstract class ManagedBlockchainTransportService<
 
         this.transportServices = this.prepareTransportServices(connections);
         this.publicTransportServices = this.transportServices.filter((ts) => !ts.getTransportConnection().isPrivate);
-        this.reference = this.publicTransportServices[0];
+
+        try {
+            this.reference = await this.getActiveTransport();
+        } catch (ex) {
+            logger.warn(
```

Comment:
throw exception after

Reply:
If exception will be thrown after, the service won't be inited and network won't be supported. but an error will be thrown if there are still no active transports, and someone will try to use this service to make a request to a blockchain node